storage/innobase/xtrabackup/doc/source/xtrabackup-files.rst

```diff
@@ -23,7 +23,7 @@ Index of files created by Percona XtraBackup
        than any external ones.
 
     * :file:`xtrabackup_checkpoints`
-       The type of the backup (e.g. full or incremental), its state (e.g.
+       The type of the backup (for example full or incremental), its state (for example
```

Comment:
for example, full or incremental
for example, prepared

Reply:
added comma